packages/animations/README.md

```diff
@@ -10,6 +10,89 @@ Currently, the following animated effects are available in this library:
 
```

Comment:
Can you remove the line above where we claim that this is under heavy development?